scripts/blant-clusters.sh

```diff
@@ -170,7 +172,6 @@ for k in "${Ks[@]}"; do
 		for(start=1; start<=FNR; start++) { # look for a cluster starting on line "start". 
 		    if(QueueLength("Q")>0){QueueDelloc("Q");QueueAlloc("Q");} #Ensure the queue is empty
 		    origin=node[start];
-		    if (origin in visited) continue;
```

Comment:
Hmm. Why is this line deleted?

Reply:
This was (unsuccessfully) trying to avoid very similar communities that come next to each other. The tradeoff was we could miss a bigger result. Now very similar communities are handled later in the code